apps/browser/src/autofill/services/autofill-browser-settings.service.ts

```diff
@@ -14,18 +13,20 @@ import { BrowserApi } from "../../platform/browser/browser-api";
   providedIn: "root",
 })
 export class AutofillBrowserSettingsService {
-  browserAutofillSettingOverridden$(browserClient: BrowserClientVendor): Observable<boolean> {
-    if (browserClient === BrowserClientVendors.Unknown) {
-      return of(false);
-    }
-
-    return from(BrowserApi.permissionsGranted(["privacy"])).pipe(
-      switchMap((granted) => {
-        if (!granted) {
-          return of(false);
-        }
-        return from(BrowserApi.browserAutofillSettingsOverridden());
-      }),
+  async isBrowserAutofillSettingOverridden(browserClient: BrowserClientVendor) {
+    return (
+      browserClient !== BrowserClientVendors.Unknown &&
+      (await BrowserApi.permissionsGranted(["privacy"])) &&
+      BrowserApi.browserAutofillSettingsOverridden()
```

Comment:
You don't need the privacy permission check anymore

```suggestion
      browserClient !== BrowserClientVendors.Unknown &&
      BrowserApi.browserAutofillSettingsOverridden()
```

Reply:
@jprusik Ah yes. Removed. ;)